quay-enterprise/configure-syslog-ng.md

```diff
@@ -0,0 +1,73 @@
+## Redirecting syslog to /dev/stdout
+
+By default, Quay Enterprise logs most relevant for debugging to `/var/log/syslog` within the container. [syslog-ng](https://en.wikipedia.org/wiki/Syslog-ng) can be configured to redirect these logs to `/dev/stdout` which will allow for collection by most logging solutions:
```

Comment:
and [syslog-ng][syslog-ng]
then add the link at the bottom of the page, after two blank lines:

[syslog-ng]: https://en.wikipedia.org/wiki/Syslog-ng